ergm.py

```diff
@@ -187,6 +201,127 @@ def sample_network(self, sampling_method="NaiveMetropolisHastings", seed_network
 
         return network
 
+
+class BruteForceERGM(ERGM):
+    """
+    A class that implements ERGM by iterating over the entire space of networks and calculating stuff exactly (rather
+    than using statistical methods for approximating and sampling).
+    This is mainly for tests.
+    """
+    # The maximum number of nodes that is allowed for carrying brute force calculations (i.e. iterating the whole space
+    # of networks and calculating stuff exactly). This becomes not tractable above this limit.
+    MAX_NODES_BRUTE_FORCE = 6
+
+    @staticmethod
+    def construct_adj_mat_from_int(int_code: int, num_nodes: int, is_directed: bool) -> np.ndarray:
```

Comment:
I would move this and `construct_int_from_adj_mat` to the utils.py file - seems like something that would be useful in other places as well